qr2/main.go

```diff
@@ -54,16 +54,19 @@ func StartServer() {
 }
 
 func handleConnection(conn net.PacketConn, addr net.Addr, buffer []byte) {
-	if buffer[0] == AvailableRequest {
-		logging.Notice("QR2", "Command:", aurora.Yellow("AVAILABLE"))
-		conn.WriteTo(createResponseHeader(AvailableRequest, 0), addr)
-		return
-	}
-
+	packetType := buffer[0]
 	sessionId := binary.BigEndian.Uint32(buffer[1:5])
+	session := sessions[sessionId]
```

Comment:
Now that you are creating a variable for the `*Session`, you might as well check if it exists just to be safe.
```go
session, ok := sessions[sessionId]
if !ok {
// Handle if the session doesn't exist
}
```

Reply:
Resolved in commit [5396677](https://github.com/WiiLink24/wfc-server/commit/5396677898ec39413ecbc46526b0cf768efa76c6).